frontend/src/dispatcher/js-messages.ts

```diff
@@ -41,9 +41,7 @@ export class UpdateInputHints extends JsMessage {
 	readonly hint_data!: HintData;
 }
 
-export class HintGroup extends Array<HintInfo> {}
```

Comment:
@Keavon how do you feel about this being a type def vs class extension. I think it makes the code a little bit easier to read being just a type

Reply:
Chrome devtools gives it a name when it's a class. But I don't have a strong preference.

Reply:
I think for continuity for the rest of the other messages I will make it a typedef. If there is a strong need in the future we can switch the rest over as well.

Reply:
Sure, that's fine.